core-services/image-mirroring/openshift/mapping_origin_4_8

```diff
@@ -252,7 +255,8 @@ registry.ci.openshift.org/origin/4.8:tools quay.io/openshift/origin-tools:4.8 qu
 registry.ci.openshift.org/origin/4.8:upi-installer quay.io/openshift/origin-upi-installer:4.8 quay.io/openshift/origin-upi-installer:4.8.0 quay.io/openshift/origin-upi-installer:latest
 registry.ci.openshift.org/origin/4.8:vertical-pod-autoscaler quay.io/openshift/origin-vertical-pod-autoscaler:4.8 quay.io/openshift/origin-vertical-pod-autoscaler:4.8.0 quay.io/openshift/origin-vertical-pod-autoscaler:latest
 registry.ci.openshift.org/origin/4.8:vertical-pod-autoscaler-operator quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.8 quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.8.0 quay.io/openshift/origin-vertical-pod-autoscaler-operator:latest
+registry.ci.openshift.org/origin/4.8:vmware-vsphere-csi-driver quay.io/openshift/origin-vmware-vsphere-csi-driver:4.8 quay.io/openshift/origin-vmware-vsphere-csi-driver:4.8.0 quay.io/openshift/origin-vmware-vsphere-csi-driver:latest
+registry.ci.openshift.org/origin/4.8:vmware-vsphere-csi-driver-operator quay.io/openshift/origin-vmware-vsphere-csi-driver-operator:4.8 quay.io/openshift/origin-vmware-vsphere-csi-driver-operator:4.8.0 quay.io/openshift/origin-vmware-vsphere-csi-driver-operator:latest
```

Comment:
Why isn't this just `vsphere-csi-driver`?  That would be more consistent naming with `vsphere-problem-detector` and i don't think there will be 2 vspheres

Reply:
For CSI drivers & operators we settled on this naming convention: `platform-pluginname-csi-driver{-operator}`. Examples of existing drivers and operators:

* openstack-cinder-csi-driver and openstack-cinder-csi-driver-operator
* aws-ebs-csi-driver and aws-ebs-csi-driver-operator
* gcp-pd-csi-driver and gcp-pd-csi-driver-operator
* azure-disk-csi-driver and azure-disk-csi-driver-operator (this is going to be added in 4.8)

Going further we'll have:

* aws-efs-csi-driver and operator
* gcp-filestore-csi-driver and operator

Unfortunately manila and ovirt don't follow this convention (they were developed by other teams).

Reply:
vmware-vsphere is not the platform name, vsphere is.  so the name of the image is incorrect.

Reply:
The naming inconsistency between CI and ART broke nightlies (ART name includes a -rhel8- suffix which it should not)

Reply:
to clarify little bit - I think originally we went for `<vendor>-<product>-csi-driver-operator` naming convention, where aws, gcp, vmware are vendor names etc. We could drop the vendor and just keep product name for vsphere.

The nightly build as I understand is broken because of selected image type and generated suffix. :/

Reply:
Images have been renamed to `vsphere-csi-*`.